src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,44 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
```

Comment:
Please set static property outside the class to pass the tests.
this way: `Animal.alive = [];`